src/main/java/ladder/control/input/InputControl.java

```diff
@@ -0,0 +1,26 @@
+package ladder.control.input;
+
+import ladder.view.View;
+import ladder.view.input.DummyView;
+
+import java.util.Scanner;
+
+public abstract class InputControl<T> {
+    protected static final Scanner scanner = new Scanner(System.in);
```

Comment:
```suggestion
    protected static final Scanner SCANNER = new Scanner(System.in);
```
입 출력에 대한 구조를 아주 잘 설계 해주신 것 같네요 👍

Reply:
예전에 읽었던 Effective Java에서 `static final`로 선언한 객체 인스턴스는 네이밍을 camelCase로 해두길래 저렇게 선언했는데, upper case로 고쳐주셔서 java 내부 모듈을 까보니 전부 UPPER_CASE 군요...! 수정해보겠습니당